tests/modelfree/policies/test_wrappers.py

```diff
@@ -0,0 +1,238 @@
+    policies = [_get_constant_policy(half_env,
+                                     constant_value=half_env.action_space.sample(),
+                                     state_shape=state_shape) for _ in range(10)]
+    if add_zoo:
+        policies += [zoo] * num_zoo
+
+    policy_wrapper = MultiPolicyWrapper(policies=policies, num_envs=num_envs)
+
+    vec_env = CurryVecEnv(venv=vec_env,
+                          policy=policy_wrapper,
+                          agent_idx=victim_index,
+                          deterministic=False)
+    vec_env = FlattenSingletonVecEnv(vec_env)
+
+    yield vec_env, policy_wrapper, zoo
+    policy_wrapper.close()
+
+
+SIMPLE_CONFIGS = [dict(env_name="CartPole-v1",
+                       num_envs=2,
+                       num_steps=1000),
+                  dict(env_name="CartPole-v1",
+                       num_envs=2,
+                       num_steps=1000,
+                       state_shape=(10,))]
+
+
+@pytest.mark.parametrize("test_config", SIMPLE_CONFIGS)
+def test_simple_multi_policy_wrapper(test_config, tmpdir):
+    env_name, num_envs, num_steps = (test_config["env_name"],
+                                     test_config["num_envs"],
+                                     test_config["num_steps"])
+    state_shape = test_config.get("state_shape", None)
+
+    with create_simple_policy_wrapper(env_name,
+                                      num_envs,
+                                      state_shape,
+                                      ) as (vec_env, policy_wrapper):
+        obs = vec_env.reset()
+        dones = np.full(shape=num_envs, fill_value=False)
+        current_policies = [ptm.policy.constant for ptm in policy_wrapper.current_env_policies]
+        num_identical = 0
+        num_switches = 0
+        for i in range(num_steps):
+            actions, states = policy_wrapper.predict(obs, mask=dones)
+            new_current_policies = [ptm.policy.constant for ptm in
+                                    policy_wrapper.current_env_policies]
+            for ind, done in enumerate(dones):
+                policies_match = new_current_policies[ind] == current_policies[ind]
+                if not done:
+                    assert policies_match
+                if done:
+                    num_switches += 1
+                    if policies_match:
+                        num_identical += 1
+
+            current_policies = new_current_policies
+            obs, rew, dones, infos = vec_env.step(actions)
+
+        dont_match_msg = "Your policies never empirically change at episode boundaries"
+        assert num_identical / num_switches != 1, dont_match_msg
+
+
+MULTI_AGENT_ZOO_CONFIGS = [dict(env_name="multicomp/YouShallNotPassHumans-v0",
+                                num_envs=2,
+                                num_steps=1000)]
+
+
+@pytest.mark.parametrize("test_config", MULTI_AGENT_ZOO_CONFIGS)
+def test_constant_and_zoo_multi_agent_policy_wrapper(test_config, tmpdir):
```

Comment:
This seems to test a subset of `test_constant_multi_agent_multi_policy_wrapper`, so why do we need both?